Mathlib/Data/Matrix/Basic.lean

```diff
@@ -2166,7 +2166,6 @@ theorem conjTranspose_ratCast_smul [DivisionRing R] [AddCommGroup α] [StarAddMo
     (c : ℚ) (M : Matrix m n α) : ((c : R) • M)ᴴ = (c : R) • Mᴴ :=
   Matrix.ext <| by simp
 
-@[simp]
```

Comment:
How does simp prove this?

Reply:
`conjTranspose_smul` + `star_trivial`

Reply:
Ah nice, it's the new `StarModule` instance that does it.

Reply:
Can you tweak the docstring on line 2092?